balancer-js/examples/spotPrice.ts

```diff
@@ -1,44 +1,67 @@
 import dotenv from 'dotenv';
-import { BalancerSDK, Network, BalancerSdkConfig } from '../src/index';
+import { BalancerSDK, Network, BalancerSdkConfig, PoolModel, BalancerError, BalancerErrorCode } from '../src/index';
 import { ADDRESSES } from '../src/test/lib/constants';
 
 dotenv.config();
 
+const network = Network.MAINNET;
+const config: BalancerSdkConfig = {
+  network,
+  rpcUrl: `https://mainnet.infura.io/v3/${process.env.INFURA}`,
+};
+
+const balancer = new BalancerSDK(config);
 
 /*
-Example showing how to use SDK to get spot price for a pair.
+Uses SDK to find spot price for pair in specific pool.
 */
-async function getSpotPrice() {
-  const network = Network.MAINNET;
-  const config: BalancerSdkConfig = {
-    network,
-    rpcUrl: `https://mainnet.infura.io/v3/${process.env.INFURA}`,
-  };
-
-  const balancer = new BalancerSDK(config);
+async function getSpotPricePool() {
 
   const wethDaiPoolId =
     '0x0b09dea16768f0799065c475be02919503cb2a3500020000000000000000001a';
-  // This will fetch pools information using data provider
+  const daiWethPool: PoolModel | undefined = await balancer.poolsProvider.find(wethDaiPoolId);
+  if (!daiWethPool) throw new BalancerError(BalancerErrorCode.POOL_DOESNT_EXIST);
+
   const spotPriceEthDai = await balancer.pricing.getSpotPrice(
```

Comment:
This example should also be `calcSpotPrice` since it's calculating for a specific pool rather than finding the most liquid path, right?

Reply:
I didn't even notice that! Thanks.